### ISSUE TITLE
Fix `keyframe-selector-notation` false positives for named timeline ranges

### ISSUE THREAD

**Issue by Mouvedia** [stylelint/stylelint#6538]
### Demand

Add tests that ensure that
- `@keyframes foo { enter 0% {} enter 100% {} }`
- `@keyframes foo { exit 0% {} exit 100% {} }`
- `@keyframes foo { cover 0% {} cover 100% {} }`
- `@keyframes foo { contain 0% {} contain 100% {} }`

are supported/ignored.

ref: https://www.w3.org/TR/scroll-animations-1/#view-timelines-ranges

All three possible values (`"keyword"`,`"percentage"`, `"percentage-unless-within-keyword-only-block"`) must be covered.

**Comment by ybiquitous** [comment]
@Mouvedia Thanks for the report. I'm unfamiliar with the Timeline Range spec, so please let me clarify.

Here's a [demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIACBrGBPAZgJwIYFsYBnAAkwghOAB0A7EkmWgFxmxIAYBSKgXzoZNW7AIwdufOv1p1UGHAWIkARrnY16nHsGkMxEnVLqy0WPIVKqAXlQFlsEfJM3NKh2tJAAaEEXQtcAA8EcCIib3AIWkwASwBzEI0GahBsAFcAG2IU+FtNZJBTBUIAWiIYLLBXbBLaCGZcZhionJIU0wB3CGwoFLtpXhBeIA):

```css
@keyframes foo {
  enter 0% {}   /* ← Expected "0%" to be "from" (Unexpected? Should ignore?) */
  enter 100% {} /* ← Expected "0%" to be "from" (Unexpected? Should ignore?) */
}

@keyframes bar {
  0% {}   /* ← Expected "0%" to be "from" (Expected 👍🏼) */
  100% {} /* ← Expected "100%" to be "to" (Expected 👍🏼) */
}

@keyframes baz {
  from {} /* ← No reported (Expected 👍🏼) */
  to {}   /* ← No reported (Expected 👍🏼) */
}
```

```json
{
  "rules": {
    "keyframe-selector-notation": "keyword"
  }
}
```

Do you mean we should fix a false positive above when specifying the `"keyword"` option?

**Comment by ybiquitous** [comment]
Thanks for the explanation. Let's ignore the `<timeline-range-name> <percentage>` notation.

I've labeled the issue as **ready to implement**. Please consider [contributing](https://stylelint.io/contributing) if you have time.

There are [steps on how to fix a bug in a rule](https://stylelint.io/developer-guide/rules#fix-a-bug-in-a-rule) in the Developer guide.
